packages/network-controller/src/NetworkController.ts

```diff
@@ -400,35 +400,40 @@ export class NetworkController extends BaseControllerV2<
     this.refreshNetwork();
   }
 
-  getEIP1559Compatibility() {
+  getLatestBlock(): Promise<Block> {
```

Comment:
Could we keep this private for now? Would rather not add to the public API if we can avoid it, because then any further changes to this become breaking changes.